docker/Dockerfile-versioned

```diff
@@ -2,9 +2,8 @@ FROM temp
 
 ARG DOCKER_VERSION=VERSION
 
-RUN add-apt-repository \
-       "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" && \
-    apt-get -y update && \
```

Comment:
Do we need to run `apt-get update` after `add-apt-repository`? If so, we need this line